docs/manual-EN/1.overview/2.quick-start/4.import-csv-file.md

```diff
@@ -0,0 +1,243 @@
+
+```ngql
+version: v1rc1
+description: example
+clientSettings:
+  concurrency: 2 # number of graph clients
+  channelBufferSize: 1
+  space: nba
+  connection:
+    user: user
+    password: password
+    address: 127.0.0.1:3699
+logPath: ./err/test.log
+files:
+  - path: /home/nebula/serve.csv
+    failDataPath: ./err/serve.csv
+    batchSize: 2
+    type: csv
+    csv:
+      withHeader: false
+      withLabel: false
+    schema:
+      type: edge
+      edge:
+        name: serve
+        withRanking: false
+        props:
+          - name: start_year
+            type: int
+          - name: end_year
+            type: int
+  - path: /home/nebula/follow.csv
+    failDataPath: ./err/follow.csv
+    batchSize: 2
+    type: csv
+    csv:
+      withHeader: false
+      withLabel: false
+    schema:
+      type: edge
+      edge:
+        name: follow
+        withRanking: false
+        props:
+          - name: degree
+            type: int
+  - path: /home/nebula/player.csv
+    failDataPath: ./err/player.csv
+    batchSize: 2
+    type: csv
+    csv:
+      withHeader: false
+      withLabel: false
+    schema:
+      type: vertex
+      vertex:
+        tags:
+          - name: player
+            props:
+              - name: name
+                type: string
+              - name: age
+                type: int
+  - path: /home/nebula/team.csv
+    failDataPath: ./err/team.csv
+    batchSize: 2
+    type: csv
+    csv:
+      withHeader: false
+      withLabel: false
+    schema:
+      type: vertex
+      vertex:
+        tags:
+          - name: team
+            props:
+              - name: name
+                type: string
+
+```
+
+**Note**:
+
+* In the above configuration file, you must change the IP address and the port number to yours.
+
+* You must change the directory of the CSV files to yours, otherwise **Nebula Importer** cannot find the CSV files.
+
+## Preparing the CSV Data
+
+In this example we prepare four CSV data files: **player.csv**, **team.csv**, **serve.csv**, and **follow.csv**.
+
+The data in the **serve.csv** file is as follows:
+
+```csv
+100,200,1997,2016
+101,201,1999,2018
+102,203,2006,2015
+102,204,2015,2019
+103,204,2017,2019
+104,200,2007,2009
+```
+
+The data in the **follow.csv** file is as follows:
+
+```csv
+100,101,95
+100,102,90
+101,100,95
+102,101,75
+102,100,75
+103,102,70
+104,101,50
+104,105,60
+105,104,83
+```
+
+The data in the **player.csv** file is as follows:
+
+```csv
+100,Tim,42
+101,Tony,36
+102,LaMarcus,33
+103,Rudy,32
+104,Marco,32
+105,Danny,31
+106,Kyle,25
+107,Aron,32
+108,Boris,36
+```
+
+The data in the **team.csv** file is as follows:
+
+```csv
+200,Warriors
+201,Nuggets
+202,Rockets
+203,Trail
+204,Spurs
+205,Thunders
+206,Jazz
+207,Clippers
+208,Kings
+```
+
+**Note**:
+
+* In the **serve** and **follow** CSV files, the first two columns are vertex IDs and the other columns are in consistent with the `config.yaml` file.
```

Comment:
Should we specify which column is the source vertex ID here?

Reply:
I do not think there is a misunderstanding for foreigners.